### DISCUSSION TITLE
Any plans to remove the default configuration options blocking PDF handling?

### DISCUSSION THREAD

**Discussion by sbraford** [ImageMagick/ImageMagick#3953] [Development]
Hello.

We are having some issues with (ImageMagick 6)'s default security options regarding PDF handling.

This StackOverflow thread details the issue: https://stackoverflow.com/questions/52998331/imagemagick-security-policy-pdf-blocking-conversion

I understand that these six lines were added to policy.xml due to a vulnerability present in Ghostscript versions earlier than 9.24.

<img width="666" alt="Screen Shot 2021-07-17 at 3 28 07 PM" src="https://user-images.githubusercontent.com/356/126050663-b641592d-b168-43fa-9ab2-f41712814d4b.png">

Are there any plans to remove these lines from the default installation?

We use ImageMagick installed via apt in a Docker container. It looks like Ghostscript is not even installed in our containers, so we should be safe to remove those lines. Our planned solution is to add this line to our Dockerfile:

```
sed -i '/disable ghostscript format types/,+6d' /etc/ImageMagick-6/policy.xml
```

Just wondering if this will always be necessary, or if there are any plans to remove these lines now that a patch and update has been available for Ghostscript for a while now.

Thanks!

**Comment by dlemstra** [comment]
We do not control these settings. These settings are added by the maintainers of the linux distribution. So we are unable to answer your question.
